bot/package.json

```diff
@@ -21,14 +21,16 @@
   "license": "MIT",
   "dependencies": {
     "actions-on-google": "^2.12.0",
+    "axios": "^0.19.2",
```

Comment:
@gutorc92, why are we using `axios`?  I've had problems with this library before. Do we really need it?

Reply:
I can't make a request to telegram api using https(node default library for requests). I spent a time getting a 400 from telegram api so I added axios to the project.

Reply:
Well, so why then it worked for me? @gutorc92